lib/assert.js

```diff
@@ -152,7 +152,8 @@ assert.deepStrictEqual = function deepStrictEqual(actual, expected, message) {
       expected,
       message,
       operator: 'deepStrictEqual',
-      stackStartFn: deepStrictEqual
+      stackStartFn: deepStrictEqual,
+      errorDiff: this === strict ? 2 : 0
```

Comment:
Can you name the different error diff modes? It’s really hard to keep track of constants `0`, `1` and `2` with distinct meanings, especially in code that spans across files.